### ISSUE TITLE
Form won't save selected Mailchimp group

### ISSUE THREAD

**Issue by leetleet** [elementor/elementor#7083]
- [ x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x ] The issue still exists against the latest stable version of Elementor.

## Description

I’m trying to create a form, that when submitted, will add the user to a specific Mailchimp group.

I have Mailchimp connected and the proper list selected, but when trying to select the group the person is added to, it won’t save. I select it, update the page, click on something else, and the selected group disappears. This happens on all the forms on my site.

Below, I have the group selected in the form, but when I click into a new tab or update and exit the editor, then come back to check, the group disappears and won't save.

<img width="1440" alt="screen shot 2019-02-12 at 11 15 41 pm" src="https://user-images.githubusercontent.com/47235377/52811583-1391f300-305b-11e9-8f79-08189e3a2bc1.png">

<img width="1440" alt="screen shot 2019-02-14 at 1 21 45 pm" src="https://user-images.githubusercontent.com/47235377/52811777-8b601d80-305b-11e9-8202-11676d92e189.png">

I’ve tried saving it multiple ways and testing it out, and it still won’t save the specified group. Any ideas why this is happening / how to fix it? Much appreciated and thank you!

(The form itself works - it's just the group aspect which is causing trouble. Want it to work so when someone signs up, they are automatically added to a campaign in Mailchimp).

## Steps to reproduce

1. Connect mailchimp account to elementor
2. Create a form
3. Specify "Mailchimp" in actions after submit
4. Select the proper mailchimp list
5. Add a group
6. Save or simply click a different tab
7. Come back to mailchimp tab to see if Group field saved

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 84.1, Revision b308619 v5.6.41-84.1
	PHP Version: 7.0.33
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.0.3
	Site URL: https://edgeofcomfort.com
	Home URL: https://edgeofcomfort.com
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 40M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/Chicago
	Debug Mode: Inactive

== Theme ==
	Name: Phlox
	Version: 2.3.5
	Author: averta
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.96 Safari/537.36

== Active Plugins ==
	Advanced Gutenberg
		Version: 1.10.1
		Author: JoomUnited

	Akismet Anti-Spam
		Version: 4.1.1
		Author: Automattic

	Blubrry PowerPress
		Version: 7.4
		Author: Blubrry

	Elementor
		Version: 2.4.6
		Author: Elementor.com

	Elementor Pro
		Version: 2.4.4
		Author: Elementor.com

	Facebook Widget
		Version: 4.2.3
		Author: Milap Patel

	Gallery - Video Gallery
		Version: 2.2.2
		Author: Huge-IT

	ImageInject
		Version: 1.16
		Author: Thomas Hoefter

	Imsanity
		Version: 2.4.2
		Author: Exactly WWW

	Jetpack by WordPress.com
		Version: 7.0
		Author: Automattic

	jQuery Pin It Button for Images
		Version: 3.0.2
		Author: Marcin Skrzypiec

	MailChimp Forms by MailMunch
		Version: 3.1.2
		Author: MailMunch

	Nomad World Map
		Version: 1.3.1
		Author: Nomad World Map

	Phlox Core Elements
		Version: 2.3.7
		Author: averta

	Really Simple SSL
		Version: 3.1.3
		Author: Rogier Lankhorst, Mark Wolters

	Simple Author Box
		Version: 2.3.0
		Author: MachoThemes

	Smart Slider 3
		Version: 3.3.13
		Author: Nextend

	Social Media Flying Icons | Floating Social Media Icon
		Version: 4.3.1
		Author: Acurax

	Stackable - Gutenberg Blocks
		Version: 1.12.1
		Author: Gambit Technologies, Inc

	Vafpress Post Formats UI
		Version: 1.5
		Author: vafpress

	WP-Optimize
		Version: 2.2.12
		Author: David Anderson, Ruhani Rabin, Team Updraft

	WPForms Lite
		Version: 1.5.1
		Author: WPForms

	WP Google Fonts
		Version: v3.1.4
		Author: Noah Kagan

	WP Instagram Widget
		Version: 2.0.3
		Author: Scott Evans

	WPtouch Mobile Plugin
		Version: 4.3.35
		Author: WPtouch

	Yoast SEO
		Version: 9.6
		Author: Team Yoast

== Must-Use Plugins ==
	Endurance Browser Cache
		Version: 0.4
		Author: Mike Hansen

	Endurance Page Cache
		Version: 1.3
		Author: Mike Hansen

	Endurance PHP Edge
		Version: 0.1
		Author: Mike Hansen

	SSO
		Version: 0.1
		Author: Garth Mortensen, Mike Hansen

== Log ==
	: 
PHP: showing 3 of 3PHP: 2019-01-28 23:54:41 [notice X 1][/home7/edgeofco/public_html/wp-content/plugins/elementor/includes/managers/image.php::33] Undefined index: id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-29 17:07:08 [warning X 4][/home7/edgeofco/public_html/wp-content/plugins/elementor-pro/modules/query-control/module.php::31] array_merge(): Argument #2 is not an array [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-01-29 17:16:33 [warning X 2][/home7/edgeofco/public_html/wp-content/plugins/elementor-pro/modules/query-control/module.php::31] array_merge(): Argument #1 is not an array [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 7 of 7JS: 2019-01-29 07:14:01 [error X 1][https://edgeofcomfort.com/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=2.4.2:2:15986] Cannot read property \'model\' of undefined 
JS: 2019-01-29 19:47:53 [error X 1][https://edgeofcomfort.com/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=2.4.2:2:20587] Cannot read property \'updateMap\' of undefined 
JS: 2019-01-29 23:16:18 [error X 1][https://edgeofcomfort.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.4.4:2:227895] Cannot read property \'statusText\' of undefined 
JS: 2019-01-31 03:12:05 [error X 101][https://edgeofcomfort.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.4.5:2:167469] Cannot read property \'errors\' of undefined 
JS: 2019-01-31 04:13:55 [error X 1][https://edgeofcomfort.com/wp-content/plugins/elementor/assets/lib/slick/slick.min.js?ver=1.8.1:1:19335] Cannot read property \'add\' of null 
JS: 2019-02-01 06:24:16 [error X 1][https://edgeofcomfort.com/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=2.4.3:2:20171] Cannot read property \'setValue\' of undefined 
JS: 2019-02-09 00:22:53 [error X 1][https://edgeofcomfort.com/wp-content/plugins/elementor/assets/lib/e-select2/js/e-select2.full.min.js?ver=4.0.6-rc.1:1:60824] Cannot read property \'query\' of null 

Log: showing 4 of 42019-01-30 19:18:51 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.4',
  'to' => '2.4.5',
)]
2019-01-30 19:19:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.2',
  'to' => '2.4.3',
)]
2019-02-12 23:09:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.5',
  'to' => '2.4.6',
)]
2019-02-12 23:12:00 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.3',
  'to' => '2.4.4',
)]

**Comment by shilo-ey** [comment]
Hi @leetleet 

We are familiar with this issue and already fixed it. 

The fix should be released in the next version of Elementor Pro.

Thanks.

**Comment by leetleet** [comment]
Terrific, thank you! Any idea when the next version will be released?

**Comment by shilo-ey** [comment]
Hi @leetleet 

This issue is fixed in Elementor Pro v2.4.5

Feel free to update!

thanks.
